app/representers/api/authorization_representer.rb

```diff
@@ -18,9 +19,12 @@ class Api::AuthorizationRepresenter < Api::BaseRepresenter
       count: represented.accounts.count
     }
   end
-  embed :accounts, paged: true, item_class: Account, item_decorator: Api::Min::AccountRepresenter, per: :all
+  embed :accounts, get: :approved_active_accounts, paged: true, per: :all,
+                   item_class: Account, item_decorator: Api::Auth::AccountMinRepresenter,
+                   url: lambda { |_r| api_authorization_accounts_path }
```

Comment:
Ah!  I didn't know such a thing existed.  Will update, and remove my changes to `embeds.rb`